packages/wonder-blocks-accordion/src/components/__tests__/accordion.test.tsx

```diff
@@ -175,20 +175,24 @@ describe("Accordion", () => {
         );
 
         // Act
-        const header1 = screen.getByTestId("test-id-1-header");
-        const header2 = screen.getByTestId("test-id-2-header");
+        const header1 = await screen.findByTestId("test-id-1-header");
+        const header2 = await screen.findByTestId("test-id-2-header");
 
         // Assert
         expect(header1).toBeVisible();
         expect(header2).toBeVisible();
     });
 
-    test("Styles the corners based on the square cornerKind", () => {
+    test("Styles the corners based on the square cornerKind", async () => {
         // Arrange
         render(
             <Accordion cornerKind="square">
                 {[
-                    <AccordionSection header="Title" testId="section-test-id">
+                    <AccordionSection
+                        key="section-id"
```

Comment:
**question:** Does `render` throw a warning about keys now? not sure if this was happening before, but you just uncovered (the warning makes sense btw).

Reply:
I'm not sure if it's new, or not! But I noticed it so decided to clean it up while I was in there...